src/dask_histogram/boost.py

```diff
@@ -34,6 +36,60 @@
 __all__ = ("Histogram", "histogram", "histogram2d", "histogramdd")
 
 
+def _hist_safe_sum(items):
+    safe_items = [item for item in items if not isinstance(item, tuple)]
```

Comment:
Could use a comment: why are there tuples and why do we drop them?

Also, a duplicate version is in core.py, below

Reply:
This was the duplicate, just use `sum` now in the tree reduction.